python/tvm/relay/frontend/pytorch.py

```diff
@@ -4622,7 +4622,7 @@ def create_convert_map(self):
             "aten::floor_divide": self.make_elemwise("floor_divide"),
             "aten::true_divide": self.make_elemwise("divide"),
             "aten::fmod": self.make_elemwise("trunc_mod"),
-            "aten::remainder": self.make_elemwise("floor_mod"),
```

Comment:
Why are we changing this from floor_mod to pure mod?

Reply:
I tested with both floor_mod and mod op and I didn't face any mismatch there. So I Will close this PR and track with floor_mod itself @nvukobratTT